dao/src/main/java/uk/gov/moj/sdt/dao/GenericDao.java

```diff
@@ -282,16 +282,23 @@ private <D extends IDomainObject> CriteriaQuery<D> createSelectQuery(Class<D> do
 
     protected Predicate createDatePredicate(CriteriaBuilder criteriaBuilder, Root<T> root, int dataRetention) {
         Path<LocalDateTime> createdDatePath = root.get("createdDate");
+        LOGGER.debug("createDatePredicate:{}", criteriaBuilder.and(
+                criteriaBuilder.greaterThanOrEqualTo((createdDatePath), atStartOfDay(dataRetention)),
+                criteriaBuilder.lessThan((createdDatePath), atBeginningOfNextDay())));
```

Comment:
Is this debug line one that was used during development? If so it may be able to be removed.